server/tests/audit_math/test_sampler.py

```diff
@@ -134,6 +134,31 @@ def test_macro_recount_sample(close_macro_batches, close_macro_contest, snapshot
     snapshot.assert_match(sample)
 
 
+def test_draw_macro_multiple_contests(macro_batches, snapshot):
+    name = "test2"
+
+    info_dict = {
+        "cand1": 400,
+        "cand2": 100,
+        "ballots": 500,
+        "numWinners": 1,
+        "votesAllowed": 1,
+    }
+
+    other_contest = Contest(name, info_dict)
+
+    for batch in macro_batches:
+        pct = int(batch.split(" ")[-1])
+        if pct < 10:
+            macro_batches[batch]["test2"] = {"cand1": 40, "cand2": 10, "ballots": 50}
+
+    # This should give us zeros for error
```

Comment:
i don't quite understand this comment. can you explain a bit more?

also not totally sure what this test is for - how does it test multiple contests?

Reply:
I will add a more explanatory comment. Basically there's a branch in macro that happens if the max possible error is 0. One way to make that happen is to compute the error for a batch that doesn't have the contest on its ballots. Here I'm adding a second contest to only some of the batches to make sure that happens.